src/OnboardingSPA/components/NewfoldInterfaceSkeleton/index.js

```diff
@@ -9,23 +9,26 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { forwardRef, useEffect } from '@wordpress/element';
+import { forwardRef, useEffect, useContext } from '@wordpress/element';
 // eslint-disable-next-line  @wordpress/no-unsafe-wp-apis
 import { __unstableUseNavigateRegions as useNavigateRegions } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { useMergeRefs } from '@wordpress/compose';
-// import bgAiImg from '../../static/images/ai_bg.png';
+// Needs to explicitly imported to be added dynamically
+import AIImg from '../../static/images/sitegen/ai_bg.png';
```

Comment:
Should we use camel case for `AIImg`? Should we rename it to `aiBg`?

Reply:
Fixed.